exporter/kafkaexporter/config.go

```diff
@@ -40,6 +41,9 @@ type Config struct {
 	// The name of the kafka topic to export to (default otlp_spans for traces, otlp_metrics for metrics)
 	Topic string `mapstructure:"topic"`
 
+	// TopicFromAttribute is the name of the attribute to use as the topic name.
+	TopicFromAttribute string `mapstructure:"topic_from_attribute"`
```

Comment:
This option should be added to the exporter's README. Also, please clarify in the description that the attribute referenced should be a resource attribute. Another thing to be very clear about is that this option will take precedent over the `topic` configuration option if specified.